### PR TITLE
fix: Remembered institutions

### PR DESCRIPTION
**Pull request by BaranekD**

### CODE REVIEW

File: www/resources/campus-idp.css [hunk 1]
```diff
@@ -67,6 +67,14 @@ html, body {
     margin-bottom: 24px;
 }
 
+.remembered-institution.remembered-institution {
```
Comment (melanger) [inline www/resources/campus-idp.css:70]:
```suggestion
.remembered-institution {
```

File: www/resources/campus-idp.css [hunk 2]
```diff
@@ -67,6 +67,14 @@ html, body {
...
+    margin-top: 12px;
+}
+
+.list-group-item + .list-group-item {
```
Comment (melanger) [inline www/resources/campus-idp.css:74]:
```suggestion
.framework_bootstrap5 .list-group-item + .list-group-item {
```

File: templates/includes/searchbox.twig [hunk 3]
```diff
@@ -40,7 +40,7 @@
                             </button>
                         </div>
                     {% else %}
-                        <button class="list-group-item list-group-item-action d-flex align-items-center" type="submit" name="searchbox" value="{{ idp.entityid }}">
+                        <button class="list-group-item list-group-item-action d-flex align-items-center rounded remembered-institution {% if configuration.priority == 'primary' %}border-dark{% else %}border-muted{% endif %}" type="submit" name="searchbox" value="{{ idp.entityid }}">
```
Comment (melanger) [inline templates/includes/searchbox.twig:43]:
```suggestion
                        <button class="list-group-item list-group-item-action d-flex align-items-center rounded remembered-institution border-{% if configuration.priority == 'primary' %}dark{% else %}muted{% endif %}" type="submit" name="searchbox" value="{{ idp.entityid }}">
```

File: www/resources/campus-idp.css (hunk 1 above)
Comment (BaranekD) [reply www/resources/campus-idp.css:70]: This does not replace the original margin-top from MUNI framework (-1px). In MUNI, this style is defined in `.box-vcards-list__item + .box-vcards-list__item`.

Comment (melanger) [reply www/resources/campus-idp.css:70]: OK, but you're tackling [CSS specifity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) (priority) by doubling a selector, that's not very clear.

You can use a different, more specific selector, or you can use [!important](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception):

```css
.remembered-institution {
    margin-top: 12px !important;
}
```

File: www/resources/campus-idp.css [hunk 4]
```diff
@@ -67,6 +67,18 @@ html, body {
     margin-bottom: 24px;
 }
 
+.framework_muni .remembered-institution {
+    margin-top: 12px !important;
+}
+
+.remembered-institution {
+    margin-top: 12px;
+}
```
Comment (melanger) [inline www/resources/campus-idp.css:76]:
```suggestion
.remembered-institution {
    margin-top: 12px !important;
}
```
`!important` can be left for Bootstrap, too (it does nothing when there are no conflicts)

**Review by melanger — APPROVED** [review]

**Pull request merged**